sdk/remoterendering/azure-mixedreality-remoterendering/samples/remote_rendering_client_sample.py

```diff
@@ -179,9 +188,9 @@ def demonstrate_rendering_session_lifecycle():
 
         # if the session should run longer than initially requested we can extend the lifetime of the session
         session = client.get_rendering_session(session_id)
-        if session.lease_time_minutes - session.elapsed_time_minutes < 2:
+        if cast(int, session.lease_time_minutes) - cast(int, session.elapsed_time_minutes) < 2:
```

Comment:
Is the `session` not typing its attributes as int here?

Reply:
Not correctly at least. Added type hints in the session type itself and works like a charm now without the cast.